components/atoms/Search/search.tsx

```diff
@@ -15,6 +16,10 @@ const Search = ({ placeholder, name, value, autoFocus, className, onSearch }: Se
     onSearch?.(search);
   };
 
+  const emptySearch = () => {
+    setSearch("");
```

Comment:
Call the provided `onSearch` function so it triggers an update for the parent component also

```suggestion
    setSearch("");
    onSearch?.("");
```